custom/icds_reports/data_pull/queries.py

```diff
@@ -22,6 +23,9 @@ def run(self, db_alias):
         cursor.copy_expert(
             "COPY ({query}) TO STDOUT DELIMITER ',' CSV HEADER;".format(query=query),
             string_buffer)
+        # dropping any setup table if created
+        if self.setup_sql_file_path and self.setup_table:
```

Comment:
same feedback here and other places where it is done.

Reply:
answered